Server/src/compile.sh

```diff
@@ -0,0 +1,35 @@
+# Author: Pontus Laestadius
+# Version: 1.0
+
+# Iterates over all erlang source files in the current directory.
+for f in *.erl; do
+	
+	# Compiles the erlang file.
+	erlc -v $f
+	
+	# Exho the terminal that it compiled.
```

Comment:
Thats not a word :D

Reply:
HAHAHAHAHAHAHAHAHHAHAHAHAHHAHAHAHHAHAH
AH
AH
AHHAHAHAHAH
HAHAHAH
AHHAHAH
HAH
AH
HA
HAHAHAHHAH
AH
HAH
AH
AH HA
HAHAHAHAHH